.github/workflows/cwltool.yml

```
@@ -23,7 +30,7 @@ jobs:
           python-version: '3.9.x'
       
       - name: Setup prerequirements
-        run: pip install cwltest cwltool
+        run: pip install cwltest cwltool udocker
```

```suggestion
        run: pip install cwltest cwltool udocker && udocker install
```

as per https://indigo-dc.github.io/udocker/installation_manual.html#23-install-from-pypi-using-pip

---

Ah! Good catch! Thanks